contracts/vaults/BaseVaults/base/RibbonVault.sol

```diff
@@ -275,26 +276,25 @@ contract RibbonVault is OptionsVaultStorage {
         Vault.DepositReceipt memory depositReceipt = depositReceipts[creditor];
 
         // If we have an unprocessed pending deposit from the previous rounds, we have to process it.
-        uint128 unredeemedShares =
+        uint256 unredeemedShares =
             depositReceipt.getSharesFromReceipt(
                 currentRound,
                 roundPricePerShare[depositReceipt.round],
                 vaultParams.decimals
             );
 
-        uint256 depositAmount = uint104(amount);
         // If we have a pending deposit in the current round, we add on to the pending deposit
         if (currentRound == depositReceipt.round) {
             uint256 newAmount = uint256(depositReceipt.amount).add(amount);
-            depositAmount = newAmount;
+            amount = newAmount;
```

Comment:
you are deleting depositAmount and instead setting 'amount'. but amount is a parameter